File: examples/python-provider/configs/fileConfig.py
"""
Simple parameters configuration - this is here to demonstrate how to load parameters configuration from a Python file
"""


def get_params():
    """
    Returns basic parameters for the API calls

    Returns:
        dict: A dictionary containing parameters
    """
    return {
        "max_tokens": 1000,
        "frequency_penalty": 0,
        "presence_penalty": 0,
        "timeout": 30,
        "foo": "bar",
    }

File: examples/python-provider/configs/fileConfig.js
/**
 * Simple formatting configuration - this is here to demonstrate how to load formatting configuration from a JavaScript file
 */
function getFormatConfig() {
  return {
    uppercase: false,
    prefix: 'Question:',
    timestamp: new Date().toISOString(),
  };
}

module.exports = {
  getFormatConfig,
};
